externs/pictureinpicture.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2016 Google Inc.
+ * Copyright 2018 Google Inc.
```

Comment:
All of our licenses have to say copyright 2016.
Because if the licenses are different, the closure compiler will copy every different version into the compiled file, which will bloat it pointlessly. Also, there might be a legal reason? I forget.